### PR TITLE
Replaced deprecated data nodes

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

**Review by tlozoot — CHANGES_REQUESTED** [review]
`browserPerf1mGroups1` has been removed completely, and is not replaced by `firewallEventsAdaptive`. Should have João Sousa Botto review and suggest changes

File: content/analytics/graphql-api/features/data-sets/index.md [hunk 1]
```diff
@@ -70,6 +69,7 @@ The following data nodes are deprecated. To avoid disruption, migrate to replace
 | `loadBalancingRequests`      | `loadBalancingRequestsAdaptive`      | September 30, 2021     |
 | `loadBalancingRequestsGroups`| `loadBalancingRequestsAdaptiveGroups`| September 30, 2021     |
 | `ipFlows1mGroups`<br/> `ipFlows1hGroups`<br/> `ipFlows1dGroups`<br/> `ipFlows1mAttacksGroups` | `spectrumNetworkAnalyticsAdaptiveGroups`<br/> `magicTransitNetworkAnalyticsAdaptiveGroups`<br/> `dosdNetworkAnalyticsAdaptiveGroups`<br/> `dosdAttackAnalyticsGroups`<br/> `flowtrackdNetworkAnalyticsAdaptiveGroups`<br/> `magicFirewallNetworkAnalyticsAdaptiveGroups` | March 31, 2022 |
+| `browserPerf1mGroups`<br/>`browserInsightsAdaptiveGroups`<br/>`webVitalsAdaptiveGroups` | `rumPerformanceEventsAdaptiveGroups`<br/>`rumPageloadEventsAdaptiveGroups`<br/>`rumWebVitalsEventsAdaptiveGroups`|  July, 2021   |
```
Comment (pedrosousa) [inline content/analytics/graphql-api/features/data-sets/index.md]: This is the only entry that doesn't have a specific day for the sunset date. Can we add one?
Also, was it really in July 2021?

File: content/analytics/graphql-api/limits/index.md [hunk 2]
```diff
@@ -50,6 +49,9 @@ Access to Network Analytics `*NetworkAnalyticsAdaptiveGroups` nodes is only avai
 | `magicTransitNetworkAnalyticsAdaptiveGroups`  |      n/a |      n/a |      n/a |    90 days |
 | `spectrumNetworkAnalyticsAdaptiveGroups`      |      n/a |      n/a |      n/a |    90 days |
 | `synAvgPps1mGroups`                           |      n/a |      n/a |      n/a |     7 days |
+| `rumPerformanceEventsAdaptiveGroups`          | 6 months | 6 months | 6 months |   6 months |
```
Comment (pedrosousa) [inline content/analytics/graphql-api/limits/index.md:52]: Currently we didn't use any units above `day` (we even had `365 days`)... Should we convert months to days here?

File: content/analytics/graphql-api/features/data-sets/index.md (hunk 1 above)
Comment (pedrosousa) [reply content/analytics/graphql-api/features/data-sets/index.md]: Maybe we could order the list by sunset date (not sure if we're following a specific order right now).

**Review by pedrosousa — APPROVED** [review]

File: content/analytics/graphql-api/features/data-sets/index.md (hunk 1 above)
Comment (angelampcosta) [reply content/analytics/graphql-api/features/data-sets/index.md]: I wasn't able to get a precise date, only month and year.

File: content/analytics/graphql-api/limits/index.md (hunk 2 above)
Comment (angelampcosta) [reply content/analytics/graphql-api/limits/index.md:52]: We also have months and 1 year in the table. I don't think we have a standard unit for all dates. Should we change this @jeehoon-cf?

Comment (pedrosousa) [reply content/analytics/graphql-api/limits/index.md:52]: Oh, we do have `6 months` already... missed it, sorry!

**Review by cf-ypark — APPROVED** [review]
LGTM

File: content/analytics/graphql-api/limits/index.md (hunk 2 above)
Comment (jeehoon-cf) [reply content/analytics/graphql-api/limits/index.md:52]: If we are to change it to days rather than months, 184 days is equivalent.

**Review by jeehoon-cf — APPROVED** [review]

**Review by tlozoot — APPROVED** [review]

**Pull request merged**